src/components/VmDetail/index.js

```diff
@@ -315,20 +315,16 @@ class VmDetail extends Component {
                     <FieldHelp content={msg.bootSequenceTooltip()} text={msg.bootSequence()} />
                   </dt>
                   <dd />
-                  <div>
-                    {
-                      vm.getIn(['os', 'bootDevices']).map((device, key) =>
-                        <React.Fragment>
-                          <dt className={style['field-shifted']}>
-                            <FieldHelp content={msg[`${sequence[key]}DeviceTooltip`]()} text={msg[`${sequence[key]}Device`]()} />
-                          </dt>
-                          <dd>
-                            {msg[`${device}Boot`]()}
-                          </dd>
-                        </React.Fragment>
-                      )
-                    }
-                  </div>
+                  {vm.getIn(['os', 'bootDevices']).map((device, key) =>
+                    <React.Fragment key={0}>
```

Comment:
Please, use unique identifier for key parameter. Also this parameter must be a string.